README.md

```diff
@@ -1,24 +1,70 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-Things you may want to cover:
+###HELLO RAILS REACT
+This project is a basic set up of a Rails project to use React and Redux created with the purpose of learnig how to add React and Redux to Rails.
 
-* Ruby version
+###Built With
+Ruby, Javascript
+Ruby on Rails, React, Redux
 
-* System dependencies
 
-* Configuration
+###Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
 
-* Database creation
+To get a local copy up and running follow these simple example steps.
 
-* Database initialization
+###Prerequisites
+To use this project you first need to have installed the following:
 
-* How to run the test suite
+Node.js
+Ruby
+PostgreSQL
+Rails
+Ruby version
+The version of ruby that is needed for this project is
 
-* Services (job queues, cache servers, search engines, etc.)
+2.7.5
 
-* Deployment instructions
+###Setup
+Once you have installed them, you need to use the following command to clone the repository:
 
-* ...
+git clone https://github.com/algerina/hello-rails-react.git
+
+###Database creation
+Next, use this command to create the databases: rake db:create
+
+Use this command to run the migrations if necesessary: rails db:migrate
+
+###Install
+To add 5 greetings into the database, first you need to access the rails console, you can do it with this command: rails c
+
+Once you are inside use this template to create the greetings:
+
+Greeting.create(message: [MESSAGE])
+
+Once you are finished, exit the console with this command: exit
+
+###Usage
+Finally, use the following command to initialize the live server:
+
+rails s
+
+
+## Author
+
+👤 **AmelKhiri**
+
+- GitHub: [@Algerina](https://github.com/Algerina)
+- LinkedIn: [Amel Khiri](https://linkedin.com/in/amel-khiri-qahwadji-37a550135)
+
+handshake Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a star if you like this project!
+
+memo License
+This project is MIT licensed.
```

Comment:
- [ ] Great job putting the information on your readme file together, please it will be nice to restructure it, as the will make your readme file more readable and professional, kindly checkout this [template](https://github.com/microverseinc/readme-template) from Microverse and follow the structure.

Reply:
Resolved, thank you for your feedback.